code/game/objects/items/trash.dm

```diff
@@ -60,7 +60,7 @@
 	icon_state = "sosjerky"
 
 /obj/item/trash/syndi_cakes
-	name = "syndi cakes"
+	name = "Nutri-Cakes"
```

Comment:
```suggestion
	name = "\improper Nutri-Cakes"
```

Reply:
The food version of Nutri-Cakes isn't labeled as improper either. Is it alright to change that in this PR or should I make a seperate PR for it?

Reply:
you can change it alongside.